src/main/java/core/basesyntax/EmployeeService.java

```diff
@@ -1,10 +1,27 @@
 package core.basesyntax;
 
+import java.util.Comparator;
 import java.util.List;
 import java.util.Set;
+import java.util.TreeSet;
 
 public class EmployeeService {
+    private Comparator<Employee> comparator = new Comparator<Employee>() {
+        @Override
+        public int compare(Employee firstEmployee, Employee secondEmployee) {
+
+            int compareAge = Integer.compare(firstEmployee.getAge(), secondEmployee.getAge());
+            if (compareAge == 0) {
+                int compareName = firstEmployee.getName().compareTo(secondEmployee.getName());
+                return compareName;
```

Comment:
Just a small hint, it can be inlined - creating a new variable is redundant in this case.